src/commands/createNewProject/dockerfileSteps/DockefileProjectLanguageStep.ts

```diff
@@ -0,0 +1,26 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.md in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import { AzureWizardPromptStep, type IAzureQuickPickItem } from "@microsoft/vscode-azext-utils";
+import { ProjectLanguage } from "../../../constants";
+import { type IDockerfileProjectContext } from "./IDockerfileProjectContext";
+
+export class DockerfileProjectLanguageStep extends AzureWizardPromptStep<IDockerfileProjectContext> {
+    public async prompt(context: IDockerfileProjectContext): Promise<void> {
+        const language: IAzureQuickPickItem<string>[] = [
+            { label: ProjectLanguage.JavaScript, data: 'node' },
```

Comment:
Should `data` be `javascript` here?

In which case would it be appropriate to use the `ProjectLanguage` enum value as the data value as well?

Reply:
Yeah it should be javascript! On the docs they also had node (javascript) not sure exactly but I'll use javascript since they use that in the examples. The problem with the `ProjectLanguage` enum value is since I am using the data property directly on this [line](https://github.com/microsoft/vscode-azurefunctions/pull/3929/files#diff-7ff0c87d39e4f588780460a9ca5278a3004ee49ae26c04b28b6b7fa1fdc0f958:~:text=await%20cpUtils.executeCommand(ext.outputChannel%2C%20nonNullValueAndProp(context%2C%20%27projectPath%27)%2C%20%22func%22%2C%20%22init%22%2C%20%22%2D%2Dworker%2Druntime%22%2C%20nonNullValueAndProp(context%2C%20%27projectLanguage%27)%2C%20%22%2D%2Ddocker%22)) using the data value won't work since they are different.